libs/shared/ui/src/lib/styles/object/typography.scss

```diff
@@ -0,0 +1,60 @@
+// Title
+h1,
+.h1 {
+  font-size: 2.25rem;
```

Comment:
why did we decide to `rem` as a measure unit for typo

Reply:
This is not good by the way in a "tailwind" concept.
Here you should rely on the helper of tailwind Remi with @apply that will add
`@apply font-3xl` for example. Thanks to this, it will rely on the value we set in the tailwind.config.ts and will autoupdate if we update the tailwind.config.ts values.
https://tailwindcss.com/docs/font-size

And REM is the unit used by tailwind so we are gonna use it too. Also for many accessibility reason, it is always better to use rem over em and over px.

Reply:
Yep, I have to fix that for the titles, I used @apply for other elements, not for him, I'll do that for others it's on my to-do